appveyor.yml

```diff
@@ -20,22 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```

Comment:
So this is off. Not sure why the re-rendering did this too us. Could you please try re-rendering again to see if it goes away? If it doesn't go away, it would be great to get some info about your setup.